docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_position: 2
+title: Margin
+hide_title: false
+---
+Derivatives allow you to place leveraged trades, meaning you ony need some of the notional value to trade on a market on Vega. Leverage is, in effect, the inverse of the initial margin that the protocol takes from your collateral when you place a trade. 
+
+Margin is the amount of assets required to keep your positions open and orders funded. You can think of margin as the 'down payment' to open a position. Leverage, meanwhile, describes how many times larger in value than your margin you can have as a position. (For example, if you need 20DAI to open a position worth 100DAI: Your leverage is 5x and your initial margin is 20% of the full value.)
+
+The exact margin requirements of open orders and positions are determined by the market's risk model and market conditions. The larger the position and the more volatile the market, the greater the amount of margin that will be set aside. The volatility tolerance of the market is driven by the risk model.
+
+## Positions
+A trader's position on a market is their open volume and orders. Margin requirements are calculated based on the position, and recalculated based on market movements and changes to the position. Orders that the increase your risk level of your position will require more margin.
+
+When a party on Vega opens a position, the minimum amount of money required to open that position is put into a margin account for that party in that market. Over the course of the position's lifetime, the margin requirements will likely change - the margin account may be topped up, and/or some margin is released back to collateral. If that party is trading on more than one market that uses the same asset, the collective positions on those markets will inform how much is set aside for margin. 
```

Comment:
I wonder if this is a good paragraph to introduce the word "initial". as in
```suggestion
When a party on Vega opens a position, the minimum amount of money required to open that position is put into a margin account for that party in that market. This is referred to as the **"initial"** margin. Over the course of the position's lifetime, the margin requirements will likely change - the margin account may be topped up, and/or some margin is released back to collateral. If that party is trading on more than one market that uses the same asset, the collective positions on those markets will inform how much is set aside for margin. 
```

Reply:
> If that party is trading on more than one market that uses the same asset, the collective positions on those markets will inform how much is set aside for margin. 
I think this might/could imply that that more margin is set aside if you have 2 positions with one key than if you had two positions with two keys.